public-ui/src/app/shared/components/rh-footer/rh-footer.tsx

```diff
@@ -1,27 +1,155 @@
-import { Component, ViewEncapsulation } from '@angular/core';
+import { Component } from '@angular/core';
 import * as React from 'react';
 
 import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
 import { menuItems, socialLinks } from 'app/shared/services/header-footer.service';
+import { reactStyles } from 'app/utils';
 import { environment } from 'environments/environment';
 
+const styles = reactStyles({
+  footer: {
+    background: 'rgba(38, 34, 98, 1)',
+    width: '100vw',
+    padding: '14px'
+  },
+  logo: {
+    display: 'grid',
+    gridTemplateColumns: '25% 25% 25% 25%',
+    width: '90%'
+  },
+  mainMenuItem: {
+    color: 'white',
+    fontSize: '0.9rem',
+    width: '100%',
+    marginTop: '0.5rem',
+    paddingBottom: '0.25rem'
+  },
+  mainMenuItemLink: {
+    color: 'white',
+    fontFamily: 'GothamBold, Helvetica Neue, sans-serif',
+    textTransform: 'uppercase'
+  },
+  nihLogo: {
+    marginTop: '1rem',
+    maxWidth: '15rem'
+  },
+  siteLogo: {
+    margin: '1rem 1rem 0 0',
+    maxWidth: '15rem'
+  }
+});
+
+const css = `
+  .foot-apply a {
+    display: block;
+    margin-top: 1rem;
+    text-align: center;
+    background: #ffffff;
+    font-size: 19px;
+    padding: 8px 0;
+    line-height: 19px;
+    font-family: GothamBold, "Helvetica Neue", sans-serif;
+    border: 1px solid #fff;
+  }
+  .foot-apply a:hover {
+    background-color: rgba(190, 225, 255, 1);
+  }
+  .footer-layout * {
+    font-family: "Gotham A", "Gotham B", "Helvetica Neue", sans-serif;
+    font-weight: normal;
+  }
+  .footer-layout {
+    display: flex;
+    justify-content: space-between;
+  }
+  .login-btn a {
+    display: block;
+    border: 1px white solid;
+    padding: 1em;
+    margin-top: 0.5rem;
+    text-align: center;
+    font-size: 0.7em;
+    color: white;
+    line-height: 1;
+    width: 100%;
+  }
+  .login-btn a:hover {
+    background-color: rgba(190, 225, 255, 1);
+    color: #262262;
+  }
+  .logo {
```

Comment:
Can this be removed ? We seem to be using style from reactstyles for this class.

Reply:
Nice catch. I ended up removing the JS style since this one changes based on screen size. Also renamed the class to `footer-logo` since it was getting styles applied to it from the `logo` class from `rh-header` component.